cmd/argo/commands/logs.go

```diff
@@ -100,7 +100,7 @@ func NewLogsCommand() *cobra.Command {
 					return
 				}
 				errors.CheckError(err)
-				fmt.Println(ansiFormat(fmt.Sprintf("%s: %s", event.PodName, event.Content), ansiColorCode(podName)))
+				fmt.Println(ansiFormat(fmt.Sprintf("%s: %s", event.PodName, event.Content), ansiColorCode(event.PodName)))
```

Comment:
Seems like these are the changes from #2595? We should probably keep them in separate PRs, no?